week-1/Homework/mandatory/2-logic-error.js

```diff
@@ -1,16 +1,16 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
+/* trim is a function so have to use dot to make it work */
 function trimWord(word) {
-  return wordtrim();
+  return word.trim();
 }
-
+/*.length is not a function is a Method so removed the parenthesis and added a semicolon at the end  */
```

Comment:
Fix is correct. However, length is a property, not a method. Method and function are both the same thing but are called different depending if the function is part of an object (using the function with a dot, or function on its own).